charts/spire/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: spire
 description: >
   A Helm chart for deploying the complete Spire stack including: spire-server, spire-agent, spiffe-csi-driver, spiffe-oidc-discovery-provider and spire-controller-manager.
 type: application
-version: 0.14.0
+version: 0.15.0
```

Comment:
```suggestion
version: 0.14.0
```

Reply:
Bumps are automated as a separate process

Reply:
We batch up a bunch of changes in main, then bump the version during release.